conda/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set data= load_setup_py_data() %}
+package:
+  name: ts-m2gui
+  version: {{ data.get('version') }}
+
+source:
+  path: ../
+
+build:
+  noarch: generic
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - run_m2gui = lsst.ts.m2gui.application:run_application
+
+requirements:
+  host:
+    - python {{ python }}
+    - pip
+    - setuptools_scm
+    - setuptools
+    - pytest-runner
+  build:
+    - python {{ python }}
+    - setuptools_scm
+    - setuptools
+    - ts-conda-build =0.3
+  run:
+    - python {{ python }}
+    - setuptools
+    - setuptools_scm
+    - ts-salobj
+    - ts-idl
```

Comment:
Please pin the versions of ts-salobj and ts-idl as follows:

    - ts-idl {{ idl_version }}
    - ts-salobj {{ salobj_version }}

Reply:
Updated in the commit: [6ba959e](https://github.com/lsst-ts/ts_m2gui/pull/18/commits/6ba959ea49c6e6095e71d3670546b4d6c7653eec). Thanks!